packages/itwin/measure-tools/package.json

```diff
@@ -100,7 +105,7 @@
     "redux": "^4.2.1",
     "rimraf": "^3.0.2",
     "source-map-loader": "^1.0.0",
-    "typescript": "~4.6.0",
+    "typescript": "~4.7.4",
```

Comment:
was this needed to get eslint working?

Reply:
I do not know if it is necessary, however this version is not supported, so it may or may not cause issues.
Output from console:
WARNING: You are currently running a version of TypeScript which is not officially supported by @typescript-eslint/typescript-estree.      

You may find that it works just fine, or you may not.

SUPPORTED TYPESCRIPT VERSIONS: >=4.7.4 <5.4.0

YOUR TYPESCRIPT VERSION: 4.6.2

Please only submit bug reports when using the officially supported version.

Reply:
that being said, currently the output is the same, 28 problems, so it appears to still work correctly at the moment